### PR TITLE
Eval metadata batch 3: Reddit, Rotten Tomatoes, SemEval 2010, Sentiment 140, SMS Spam, Snips, SQuAD, SQuAD v2, Timit ASR

### PR DESCRIPTION
**Pull request by sashavor**
Adding evaluation metadata for:
- Reddit
- Rotten Tomatoes
- SemEval 2010
- Sentiment 140
- SMS Spam
- Snips
- SQuAD
- SQuAD v2
- Timit ASR

### CODE REVIEW

**Comment by sashavor** [comment]
Summary of CircleCI errors:

- **sem_eval_2010_task_8**: missing 8 required positional arguments: 'annotations_creators', 'language_creators', 'licenses', 'multilinguality', 'size_categories', 'source_datasets', 'task_categories', and 'task_ids'
- **sms_spam**: `Data Instances` and`Data Splits` are empty.
- **Quora** : Expected some content in section `Citation Information` but it is empty,  missing 8 required positional arguments: 'annotations_creators', 'language_creators', 'licenses', 'multilinguality', 'size_categories', 'source_datasets', 'task_categories', and 'task_ids'
- **sentiment140**:  missing 8 required positional arguments: 'annotations_creators', 'language_creators', 'licenses', 'multilinguality', 'size_categories', 'source_datasets', 'task_categories', and 'task_ids'

There are also some timeout errors, I don't really understand the source though :confused:

File: datasets/quora/README.md [hunk 1]
```diff
@@ -3,6 +3,54 @@ languages:
 - en
 paperswithcode_id: null
 pretty_name: quora
+train-eval-index:
+- config: default
+  task: question-answering
```
Comment (lhoestq) [inline datasets/quora/README.md:8]:
```suggestion
  task: text-classification
```

**Review by lhoestq — APPROVED** [review]
Thanks !

The timeout error is unrelated to this PR, you can ignore it

File: datasets/quora/README.md [hunk 2]
```diff
@@ -3,6 +3,54 @@ languages:
 - en
 paperswithcode_id: null
 pretty_name: quora
```
Comment (lewtun) [inline datasets/quora/README.md:5]: This dataset has nested inputs which isn't supported in AutoTrain. I think we should exclude it in this first pass

File: datasets/reddit/README.md [hunk 3]
```diff
@@ -19,6 +19,32 @@ task_categories:
...
+train-eval-index:
+- config: default
+  task: summarization
+  task_id: summarization
+  splits:
+    train_split: train
+  col_mapping:
+    content: text
+    summary: target
+  metrics:
```
Comment (lewtun) [inline datasets/reddit/README.md:31]: Same comment as in other PRs about combining this as a single ROUGE metric

File: datasets/rotten_tomatoes/README.md [hunk 4]
```diff
@@ -19,6 +19,55 @@ size_categories:
...
+    train_split: train
+    eval_split: test
+  col_mapping:
+    text: text
+    label: target
+  metrics:
+    - type: accuracy
+      name: Accuracy
+    - type: f1
+      name: F1 macro
```
Comment (lewtun) [inline datasets/rotten_tomatoes/README.md:36]:
```suggestion
      name: F1
```

File: datasets/rotten_tomatoes/README.md [hunk 5]
```diff
@@ -19,6 +19,55 @@ size_categories:
...
+      args:
+        average: macro
```
Comment (lewtun) [inline datasets/rotten_tomatoes/README.md:38]: For binary classification datasets, I'm not sure it makes much sense to compute macro / micro / weighted F1 scores. Instead we can just use binary and drop the rest of the entries below - WDYT?

```suggestion
        average: binary
```

File: datasets/squad/README.md [hunk 6]
```diff
@@ -20,6 +20,19 @@ task_categories:
...
 paperswithcode_id: squad
+train-eval-index:
+- config: plain_text
+  task: question-answering
+  task_id: extractive_question_answering
+  splits:
+    train_split: train
+    eval_split: validation
+  col_mapping:
+    question: question
```
Comment (lewtun) [inline datasets/squad/README.md:31]:
```suggestion
    question: question
    context: context
```

File: datasets/squad/README.md [hunk 7]
```diff
@@ -20,6 +20,19 @@ task_categories:
...
+    answers.text: answers.text
```
Comment (lewtun) [inline datasets/squad/README.md:32]:
```suggestion
    answers.text: answers.text
    answers.answer_start: answers.answer_start
```

File: datasets/squad_v2/README.md [hunk 8]
```diff
@@ -20,6 +20,19 @@ task_ids:
...
 paperswithcode_id: squad
+train-eval-index:
+- config: squad_v2
+  task: question-answering
+  task_id: extractive_question_answering
+  splits:
+    train_split: train
+    eval_split: validation
+  col_mapping:
+    question: question
```
Comment (lewtun) [inline datasets/squad_v2/README.md:31]:
```suggestion
    question: question
    context: context
```

File: datasets/squad_v2/README.md [hunk 9]
```diff
@@ -20,6 +20,19 @@ task_ids:
...
+    answers.text: answers.text
```
Comment (lewtun) [inline datasets/squad_v2/README.md:32]:
```suggestion
    answers.text: answers.text
    answers.answer_start: answers.answer_start
```

File: datasets/squad_v2/README.md [hunk 10]
```diff
@@ -20,6 +20,19 @@ task_ids:
...
+  metrics:
+  - type: squad_v2
+    name: SQuAD_v2
```
Comment (lewtun) [inline datasets/squad_v2/README.md:35]: nit:
```suggestion
    name: SQuAD v2
```

**Review by lewtun — APPROVED** [review]
Thanks for this batch of metadata too @sashavor 🥳 ! It's looking great and I left a few minor suggestions / nits

File: datasets/rotten_tomatoes/README.md (hunk 5 above)
Comment (sashavor) [reply datasets/rotten_tomatoes/README.md:38]: sure!

**Pull request merged**